omero/sysadmins/version-requirements.rst

```diff
@@ -702,7 +758,7 @@ If no version is provided, a suitable repository is indicated.
       - Yes
     * - 3.7
       - 7 (`zeroc <https://zeroc.com/distributions/ice/>`__)
-      - 16.04 (`zeroc <https://zeroc.com/distributions/ice/>`__)
+      - 16.04, 18.04 (`zeroc <https://zeroc.com/distributions/ice/>`__)
```

Comment:
`https://zeroc.com/distributions/ice/` goes to the 3.7 downloads. This is fine for this line, but the 3.6 links in the previous line will need changing.